src/components/UserCard.jsx

```diff
@@ -1,21 +1,44 @@
-import profile from "../assets/images/profile.png";
 import messages from "../assets/icons/messages.svg";
-function UserCard() {
+
+function UserList({ user }) {
   return (
     <>
-      <div className="border-grayscale-40 flex h-[168px] w-[154px] flex-col gap-[32px] rounded-[16px] border p-[16px]">
-        <div className="flex flex-col gap-[12px]">
-          <img className="h-[48px] w-[48px]" src={profile}></img>
-          <div className="text-body2 font-regular">이름</div>
-        </div>
-        <div className="flex">
-          <img className="h-[16px] w-[16px]" src={messages}></img>
-          <span className="text-caption1 font-regular flex-1">받은 질문</span>
-          <span className="text-caption1 font-regular">9개</span>
+      <div className="flex flex-col gap-12">
+        <img className="h-48 w-48 rounded-full" src={user.imageSource} />
+        <p className="text-body2 font-regular">{user.name}</p>
+      </div>
+      <div className="flex items-center justify-between">
+        <div className="flex items-center justify-center gap-4">
+          {/* messages icon 색상 변경 필요 */}
+          <img className="h-16 w-16" src={messages}></img>
```

Comment:
```suggestion
          <img className="size-16" src={messages}></img>
```
[제안] 이렇게도 쓸 수 있으니 한 번 고려해 보시면 좋을 것 같습니다!